### DISCUSSION TITLE
R#019 节奏转换

### DISCUSSION THREAD

**Discussion by giscus** [lc4t/lc4t.github.io#27] [Announcements]
# R#019 节奏转换

2024.10.08 ~ 2024.10.14

https://blog.sakanano.moe/journals/random_019/

**Comment by wuTong0369** [comment]
follow鼓励订阅事件好的事情,只有经历大量订阅源的洗礼留下来的才是真的

**Reply by lc4t** [reply]
是个好事，就怕这样洗下来，前排全被R18占领了...
